src/test/java/core/basesyntax/service/impl/AddOperationTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitRecordDto;
+import core.basesyntax.service.FruitOperation;
+import core.basesyntax.service.operations.OperationType;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AddOperationTest {
+    private static FruitOperation operation;
+    private static FruitRecordDto fruitRecordDtoTest;
+    private static Storage storage;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        operation = new AddOperation();
+        fruitRecordDtoTest = new FruitRecordDto(OperationType.b,"banana",10);
+        storage = new Storage();
+    }
+
+    @AfterClass
```

Comment:
you should have such structure: `Before` -> `Test` -> `After`. it's about the methods order